lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -276,6 +277,11 @@ def void(authorization, options = {}, deprecated = {})
         commit(order, :void, options[:retry_logic], options[:trace_number])
       end
 
+      def allow_0_auth?(credit_card, options = {})
```

Comment:
❓  Question:

Why passing the options hash if you are not using it inside the method ?

Reply:
Thx, you're righ, fixed